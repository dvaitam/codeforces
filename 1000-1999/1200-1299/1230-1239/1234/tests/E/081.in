10 10
10 3 2 3 1 3 4 5 1 10
