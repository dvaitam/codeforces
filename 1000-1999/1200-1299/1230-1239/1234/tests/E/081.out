28 26 30 30 26 26 26 26 26 16
